src/main/kotlin/no/nav/helsearbeidsgiver/bro/sykepenger/db/ForespoerselDao.kt

```diff
@@ -243,22 +244,31 @@ class ForespoerselDao(private val db: Database) {
         }
 }
 
-fun tilForespoerselDto(row: ResultRow): ForespoerselDto =
-    ForespoerselDto(
+fun tilForespoerselDto(row: ResultRow): ForespoerselDto {
+    val orgnr = row[ForespoerselTable.orgnr].let(::Orgnr)
+    val bestemmendeFravaersdager = row[ForespoerselTable.bestemmendeFravaersdager]
+
+    val skjaeringstidspunkt =
```

Comment:
Dette er vel for bakoverkompabilitet / ingen synlig endring, men er det kanskje litt rart å regne ut denne her? 
I hvert fall når verdien som returneres kan bli forskjellig fra verdien med samme navn i databasen...

Reply:
I en perfekt verden så ville vi endret på radene i databasen, men det har vi ikke dataene til. Da tenkte jeg at det neste naturlige alternativet var å late som vi har endret på databaseradene og la DAO-en svare som om vi har det. Da holder vi endringen nærest mulig kilden.